src/components/application_manager/test/commands/hmi/simple_requests_to_hmi_test.cc

```diff
@@ -131,12 +141,12 @@ TEST_F(RequestToHMITest, SendRequest_SUCCESS) {
   SharedPtr<commands::RequestToHMI> command(
       CreateCommand<commands::RequestToHMI>());
 
-  EXPECT_CALL(mock_app_manager_, SendMessageToHMI(NotNull()));
+  EXPECT_CALL(app_mngr_, SendMessageToHMI(NotNull()));
```

Comment:
@wolfylambova22 Do you use here real instance of application manager? If not - this change is not good